packages/command-palette/src/use-command-palette.tsx

```diff
@@ -93,6 +90,13 @@ interface SiteToActionParameters {
 	};
 }
 
+function isCustomDomain( siteSlug: string | null | undefined ): boolean {
```

Comment:
Can we move this to utils or helper? I can't remember the exact name. I'm going to need this later on.

Reply:
https://github.com/Automattic/wp-calypso/pull/87890/commits/0af99a1633d64f91773c5c581f3d64a2a1c36b00